libs/sdk-utils/multicall/src/index.ts

```diff
@@ -0,0 +1,23 @@
+export function sdkUtilsMulticall(): string {
+    return 'sdk-utils-multicall';
+  }
+  
+  
+  /**
+   * MyType extends a `thing` that does some `stuff`
+   */
+  export interface MyType {
```

Comment:
Do we need this?

Reply:
will rebase hold on

Reply:
No we can delete, it's placeholder that assumes the init branch has been merged